Formula/manticoresearch.rb

```diff
@@ -70,7 +76,7 @@ def post_install
   end
 
   service do
-    run [opt_bin/"searchd", "--config", etc/"manticore/manticore.conf", "--nodetach"]
+    run [opt_bin/"searchd", "--config", etc/"manticoresearch/manticore.conf", "--nodetach"]
```

Comment:
If this changed now, maybe we should move this in a postinstall block since otherwise people suddenly use a different config.

Reply:
This just reflects actual path to the file. It was always wrong in the formula (i.e. never works at all).

Reply:
However it may be left as is.
Just - if run 'searchd' without params, it loads file with hard-coded path `<homebrew>/etc/manticoresearch/manticore.conf`. I guess, explicitly defined path in service is also expected to refer to the same file, however it could be different.